src/js/heuristicblocking.js

```diff
@@ -125,12 +127,115 @@ HeuristicBlocker.prototype = {
       return {};
     }
 
-    // ignore if there are no tracking cookies
-    if (!hasCookieTracking(details, origin)) {
+    // check if there are tracking cookies
+    if (hasCookieTracking(details, origin)) {
+      this._recordPrevalence(fqdn, origin, tabOrigin);
       return {};
     }
 
-    this._recordPrevalence(fqdn, origin, tabOrigin);
+    let self = this;
+
+    // check if this request is a cookie-syncing pixel
+    chrome.cookies.getAll({}, function(cookies) {
+      self.pixelCookieSyncAccounting(details, cookies);
+    });
+  },
+
+  /**
+   * Checks for cookie syncing: requests to third-party domains that include
+   * high entropy data from first-party cookies.
```

Comment:
We should probably note the current assumptions/limitations/future areas for exploration.